README.md

```diff
@@ -24,6 +24,16 @@ See the full documentation at [docs.openrewrite.org](https://docs.openrewrite.or
 3. Commit & push changes.
 4. Repeat periodically as new minor versions of Spring Boot are released.
 
+## Why do artifact scanners detect vulnerabilities in recipe artifacts/JARs?
+
+In order to modernize and upgrade old or vulnerable code, some OpenRewrite recipe modules bundle copies of old libraries. Libraries bundled into recipe modules are never executed. That being said, these libraries are never executed.
```

Comment:
The last two sentences end in `never executed`. Can we rewrite that without that duplication?

Reply:
while I agree I just copy pasted the docs here:
https://docs.moderne.io/administrator-documentation/moderne-platform/references/faq#why-do-artifact-scanners-detect-vulnerabilities-in-recipe-artifacts-jars

and rewrote a little as this is in the rewrite-spring context.

Reply:
I removed 1 and added a little more text behind another. Hope thats better?